### PR TITLE
4 testcases of Cephfs client feature are added

### PR DESCRIPTION
**Pull request by yogesh-mane**
ymane@localhost.localdomain
Signed-off-by: ymane ymane@localhost.localdomain

Signed-off-by: ymane ymane@localhost.localdomain ymane@redhat.com

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573877 https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573876 https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573875 https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573869 
log - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2XM47H

### CODE REVIEW

**Comment by yogesh-mane** [comment]
Changes suggested in https://github.com/red-hat-storage/cephci/pull/1034 has been done in this pull request.

File: tests/cephfs/client_mds_restriction.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        log.info(f"Verifying mount failure for client2 for {fs1}")
+        if rc == 1:
+            log.error(f"Mount success on {fs1} with client2")
+            return 1
+        return 0
+
+    except CommandFailed as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
```
Comment (AmarnatReddy) [inline tests/cephfs/client_mds_restriction.py:126]: Can we remove the commandFailed except block.
As we are covering same in generic except block

File: tests/cephfs/test_no_data_sharing_multifs.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        log.info(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Cleaning up the system")
+        client1.exec_command(sudo=True, cmd=f"rm -rf {mount_points[0]}/*")
+        client1.exec_command(sudo=True, cmd=f"rm -rf {mount_points[2]}/*")
+        client1.exec_command(sudo=True, cmd="rm -rf /home/cephuser/*backup/")
+        for client in clients:
+            for mount_point in mount_points:
+                client.exec_command(sudo=True, cmd=f"umount {mount_point}")
```
Comment (psathyan) [inline tests/cephfs/test_no_data_sharing_multifs.py:181]: Lets add `check_ec=False`

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**